packages/scripts/src/setup-prisma.ts

```diff
@@ -0,0 +1,30 @@
+import chalk from 'chalk';
+import dedent from 'dedent';
+import { writeFileSync } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+try {
+  const __filename = fileURLToPath(import.meta.url);
+  const monorepoRoot = path.join(__filename, '../../../..');
+  const pathname = path.join(monorepoRoot, 'packages/db/schema.prisma');
+
+  const content = dedent`
+    generator client {
+      provider = "prisma-client-js"
+    }
+
+    datasource db {
+      provider = "postgresql"
+      url      = env("DATABASE_URL")
+    }\n
+  `;
+
+  writeFileSync(pathname, content);
```

Comment:
Created this script to automatically create the `prisma.schema` file, since we're ignoring it from Git!